sunpy/coordinates/tests/test_transformations.py

```diff
@@ -331,9 +331,10 @@ def test_hgc_hgc():
     old = SkyCoord(90*u.deg, 10*u.deg, 1*u.AU, frame=HeliographicCarrington(obstime=obstime))
     new = old.transform_to(HeliographicCarrington(obstime=obstime + 1*u.day))
 
-    assert_quantity_allclose(new.lon, old.lon - 14.1844*u.deg, atol=1e-4*u.deg)  # solar rotation
-    assert_quantity_allclose(new.lat, old.lat, atol=1e-4*u.deg)
-    assert_quantity_allclose(new.radius, old.radius, atol=1e-5*u.AU)
+    assert_quantity_allclose(new.lon, 76.82974*u.deg, atol=1e-4*u.deg)  # solar rotation
```

Comment:
I think this changes, because the change in longitude is not (and should not) be exactly `14.1844*u.deg`, but this should be checked.